### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by rockstardev**
Fixing that console error when customer email is updated, and then also addressing #320

### CODE REVIEW

**Comment by pavlenex** [comment]
Also closes #694

**Comment by rockstardev** [comment]
I'm rarely happier then when I close issue opened by @pavlenex <3

File: BTCPayServer/Views/Invoice/Checkout.cshtml [hunk 1]
```diff
@@ -51,9 +51,9 @@
 
             <p>Alternatively, click below to continue to our HTML-only invoice.</p>
 
-            <form action="/invoice-noscript" method="GET">
-                <button style="text-decoration: underline; color: blue">Continue to javascript-disabled invoice &gt;</button>
-            </form>
+            <a href="/invoice-noscript?id=@Model.InvoiceId" style="text-decoration: underline; color: blue">
```
Comment (Kukks) [inline BTCPayServer/Views/Invoice/Checkout.cshtml]: better to use `asp-action` and `asp-route-id` :)

File: BTCPayServer/Controllers/InvoiceController.UI.cs [hunk 2]
```diff
@@ -213,6 +213,23 @@ private string ToString(PaymentMethodId paymentMethodId)
...
+            //Keep compatibility with Bitpay
+            invoiceId = invoiceId ?? id;
+            id = invoiceId;
+            //
+
+            var model = await GetInvoiceModel(invoiceId, paymentMethodId == null ? null : PaymentMethodId.Parse(paymentMethodId));
+            if (model == null)
+                return NotFound();
+
+            return View(nameof(CheckoutNoScript), model);
```
Comment (Kukks) [inline BTCPayServer/Controllers/InvoiceController.UI.cs]: I don't think you need to define the view name, it automatically uses the action method's name if non is specified

Comment (rockstardev) [reply BTCPayServer/Controllers/InvoiceController.UI.cs]: I got to that point by experimenting with View names... thinking to extract public parts of Invoice controller to CheckoutController. Thanks for suggestion, fixed it in commit b4b69394983bf1ee0b5771ff91953a1d9056c15c

File: BTCPayServer/Views/Invoice/Checkout.cshtml (hunk 1 above)
Comment (rockstardev) [reply BTCPayServer/Views/Invoice/Checkout.cshtml]: Tried it... ended up prefering single `href` element than `asp-action`, `asp-controller` and `asp-route-id`

**Pull request merged**